File: ANGULARPROJECTS/practica_angular01/src/app/Models/personaje.ts
export default class Personaje{
    public idPersonaje : number
    public idSerie : number
    public nombre : string
    public imagen : string
    

    constructor(idPersonaje ?: number , idSerie ?: number , nombre ?: string , imagen ?: string , puntuacion ?: number ,  anyo ?: Date){
        this.idSerie = idSerie ?? NaN;
        this.nombre = nombre ?? '';
        this.imagen = imagen ?? '';
        this.idPersonaje = idPersonaje ?? NaN;
    }
}

File: .NET/ASP MVC/WebApplicationPersonasMvc/WebApplicationPersonasMvc/Models/Personaje.cs
using System.ComponentModel.DataAnnotations;

namespace WebApplicationPersonasMvc.Models
{
    public class Personaje
    {
        [Required]
        public int IDPERSONAJE { get; set; }
        [Required]
        public string? Nombre { get; set; }
        [Required]
        public string? Imagen { get; set; }
    }
}
